data/mods/Russ_Extra_Stuff/items/bee_items.json

```diff
@@ -0,0 +1,80 @@
+[{
+	"type":"GENERIC",
+        "id": "bee_skep",
+        "symbol": "#",
+        "color": "brown",
+        "name": "bee skep",
+        "description": "A manmade house for bees.  Use the construction menu to start an active hive, as long as you have a queen in your inventory.  You can take apart an active hive by s)mashing, you will get your queen bee, skep, some honey and wax.  Also, build these outdoors because when you smash them, because the tile reverts to dirt.",
```

Comment:
That description is awkward. That "smashing" part shouldn't be there at all, same for the reverting to dirt. Just "Use the construction menu to build a hive" should be enough for the tips about mechanics.